packages/components/src/Tooltip/Tooltip.tsx

```diff
@@ -68,6 +71,25 @@ export default function Tooltip({
   className,
   ...rest
 }: TooltipProps) {
+  const hideOnEsc: Plugin = {
```

Comment:
nit: let's leave a comment & link to the tippy documentation:
```suggestion
  // Hides tooltip when escape key is pressed, following:
  // https://atomiks.github.io/tippyjs/v6/plugins/#hideonesc
  const hideOnEsc: Plugin = {
```

Reply:
Added comment as recommended, thanks